vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -359,7 +359,12 @@ static void startServer(Main server) throws SQLException {
                     "/api/load",
                     monitor(
                         new BlockingHandler(
-                            JsonPost.parse(MAPPER, UnloadedData.class, server::load))))
+                            JsonPost.parse(MAPPER, UnloadedData.class, (exchange, data) -> server.load(exchange, data, true)))))
+                .post(
+                    "/api/load-unverified",
```

Comment:
What do you think about `/api/load-injected` ? 

It's only the workflow run hash ID that gets skipped - all the rest of the data still gets verified. `load-injected` is clearer on the intent about when to use this endpoint.